extensions/amp-reach-player/amp-reach-player.md

```diff
@@ -20,13 +20,12 @@ limitations under the License.
 <table>
   <tr>
     <td width="40%"><strong>Description</strong></td>
-    <td>
-    An <code>amp-reach-player</code> component displays the Reach Player configured in the <a href="http://beachfrontreach.com">Beachfront Reach</a> platform.
+    <td>Displays the Reach Player configured in the <a href="http://beachfrontreach.com">Beachfront Reach</a> platform.
     </td>
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td>Beta</td>
+    <td>Stable</td>
```

Comment:
OK, fine for now. In general, I'm a fan of keeping our docs as lean as possible, looking forward to brainstorm with you soon.